grpc/src/main/java/com/linecorp/armeria/server/grpc/UnframedGrpcErrorHandler.java

```diff
@@ -48,6 +50,10 @@ static UnframedGrpcErrorHandler of(UnframedGrpcStatusMappingFunction statusMappi
         return UnframedGrpcErrorHandlers.of(statusMappingFunction);
     }
 
+    static UnframedGrpcErrorHandler of(UnframedGrpcStatusMappingFunction statusMappingFunction, MessageMarshaller jsonMarshaller) {
```

Comment:
Our convention is to let the users use the builder API for optional properties, unless it's going to used very often. The marshaller is an optional property with default values, so we probably don't need to add this method? The same rule applies to the `of*()` methods you added in this PR.